src/pages/keystone-test.tsx

```diff
@@ -0,0 +1,394 @@
+import { useState, useEffect } from 'react'
+import { Button, Grid, Alert, IconInfo } from '@trussworks/react-uswds'
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome'
+import classnames from 'classnames'
+import { useRouter } from 'next/router'
+import type { ObjectId } from 'bson'
+import { gql, useQuery } from '@apollo/client'
+import type {
+  BookmarkRecords,
+  CollectionRecords,
+  BookmarkRecord,
+  NewBookmarkInput,
+  MySpaceWidget,
+  Collection as CollectionType,
+} from 'types/index'
+
+import { WIDGET_TYPES, MAXIMUM_COLLECTIONS } from 'constants/index'
+import { withDefaultLayout } from 'layout/DefaultLayout/DefaultLayout'
+import Loader from 'components/Loader/Loader'
+import Flash from 'components/util/Flash/Flash'
+import LoadingWidget from 'components/LoadingWidget/LoadingWidget'
+import Collection from 'components/Collection/Collection'
+import Bookmark from 'components/Bookmark/Bookmark'
+import BookmarkList from 'components/BookmarkList/BookmarkList'
+import SelectableCollection from 'components/SelectableCollection/SelectableCollection'
+import Tooltip from 'components/Tooltip/Tooltip'
+import styles from 'styles/pages/sitesAndApplications.module.scss'
+
+import { useUser } from 'hooks/useUser'
+import { useMySpaceQuery } from 'operations/queries/getMySpace'
+import { useAddCollectionsMutation } from 'operations/mutations/addCollections'
+import { useAddBookmarkMutation } from 'operations/mutations/addBookmark'
+import { useAddCollectionMutation } from 'operations/mutations/addCollection'
+import { useAnalytics } from 'stores/analyticsContext'
+
+type SortBy = 'SORT_TYPE' | 'SORT_ALPHA'
+
+type SelectedCollections = string[]
+
+/** Type guards */
+function isCollection(widget: MySpaceWidget): widget is CollectionType {
+  return widget.type === WIDGET_TYPES.COLLECTION
+}
+
+const KeystoneTest = () => {
+  const router = useRouter()
+  const { user } = useUser()
+  const { loading, error, data } = useMySpaceQuery()
+  const { trackEvent } = useAnalytics()
+
+  const [sortBy, setSort] = useState<SortBy>('SORT_TYPE')
+  const [selectMode, setSelectMode] = useState<boolean>(false)
+  const [selectedCollections, setSelectedCollections] =
+    useState<SelectedCollections>([])
+  const [flash, setFlash] = useState<React.ReactNode>(null)
+
+  const [handleAddCollections] = useAddCollectionsMutation()
+  const [handleAddCollection] = useAddCollectionMutation()
+  const [handleAddBookmark] = useAddBookmarkMutation()
+  const {
+    loading: cmsCollectionsLoading,
+    error: cmsCollectionsError,
+    data: cmsCollections,
+  } = useQuery(GET_KEYSTONE_COLLECTIONS, {
+    context: {
+      clientName: 'cms',
+    },
+  })
+
+  const collections = cmsCollections?.collections as CollectionRecords
+
+  const {
+    loading: cmsBookmarksLoading,
+    error: cmsBookmarksError,
+    data: cmsBookmarks,
+  } = useQuery(GET_KEYSTONE_BOOKMARKS, {
+    context: {
+      clientName: 'cms',
```

Comment:
@jcbcapps this is the code that tells the app which API to hit -- we'll need to pass in this context with `clientName: 'cms'` with any query or mutation that we're sending to keystone.

For internal api calls, we don't need to pass in the context (it'll default to the portal api).